### PR TITLE
Add a first behavioral test

### PR DESCRIPTION
**Pull request by koushik-ms**
Contribution from the Mob Programming OSS Meetup sessions held on the 8th of September 2021 and 20th October 2021.

### CODE REVIEW

File: src/windows-gce-instance.js [hunk 1]
```diff
@@ -2,7 +2,66 @@
...
+        },
+      ],
+    },
+  };
+  console.log("Returns: ", config.os);
+  return {
+    config: {
+      os: config.os
+     }
+     } ;
```
Comment (haraldreingruber) [inline src/windows-gce-instance.js]: Is this a temporary change that should be reversed, or an intermediate step until all the required tests for this function are added?

**Review by haraldreingruber — CHANGES_REQUESTED** [review]
Thanks a lot, everybody for this contribution, I really like the approach you took. 🎉

Right now I am getting a `UnhandledPromiseRejectionWarning: Error: Invalid value for field 'resource.disks': ''. No disks are specified.` when running the command on this branch.

If the idea is to add more tests and step by step making it work again that's fine too of course, Otherwise, it would be great if the previously working behavior could be restored 🙂

Thanks for the great input.

File: src/windows-gce-instance.js (hunk 1 above)
Comment (koushik-ms) [reply src/windows-gce-instance.js]: It was an intermediate step and thanks to the mob from the session on 20.10 we now have a working version that makes it fully working again (with tests passing, of course).

**Review by haraldreingruber — APPROVED** [review]
Better late than never. Thanks for the contribution!

**Pull request merged**